migrate/pkg/migrate/tenant/tenant.go

```diff
@@ -10,14 +10,15 @@ import (
 )
 
 type App struct {
-	Name        string
-	DatabaseURL string
+	Name           string
+	DatabaseURL    string
+	DatabaseSchema string
 }
 
 func GetMigrateApps(hostnameOverride string, db sq.BaseRunner, appFilterKey string, appFilterValue string) ([]App, error) {
 	// get apps for migration
 	configSchema := "app_config"
-	sqlizer := sq.Select("app.name", "config.config #> '{app_config,database_url}'").
+	sqlizer := sq.Select("app.name", "config.config #> '{app_config,database_url,database_schema}'").
```

Comment:
This seems incorrect. `column #> '{a,b}'` roughly means the JSON expression `column["a"]["b"]`. `database_schema` is not a property of `database_url`. I guess what we want here is

```go
sq.Select("app.name",
"config.config #> '{app_config,database_url}'",
"config.config #> '{app_config,database_schema}'")
```

This will select 3 columns.